src/main/java/com/soen343/gms/service/JobService.java

```diff
@@ -0,0 +1,13 @@
+package com.soen343.gms.service;
+
+import com.soen343.gms.model.Job;
+import com.soen343.gms.model.Task;
```

Comment:
Doesn't seem like Task import is required? Task is imported in Job; thus, JobService uses Job which Job uses Task.